mathoxide-lib/src/array.rs

```diff
@@ -22,11 +70,14 @@ mod test {
 
     #[test]
     fn it_works() {
-        let array = Array::<ThreadSafeStorage<u32>>::zeros(10);
+        let array = Array::<ThreadSafeStorage<u32>, views::SimpleView>::zeros(&[4, 5]);
+        println!("{}", array);
     }
```

Comment:
this test can probably be removed all together. The `zeros` constructor is tested in `check_2d_nrows_format` and the `println!` call doesn't add anything.